README.md

```diff
@@ -134,37 +122,6 @@ except NumbersException as err:
 
 For handling all possible exceptions thrown by this SDK use `SinchException` (superclass of all Sinch exceptions) form `sinch.core.exceptions`.
 
+## License
 
-## Logging
-
-Logging configuration for this SDK utilizes following hierarchy:
-1. If no configuration was provided via `logger_name` or `logger` configurable, SDK will inherit configuration from the root logger with the `Sinch` prefix.
-2. If `logger_name` configurable was provided, SDK will use logger related to that name. For example: `myapp.sinch` will inherit configuration from the `myapp` logger.
-3. If `logger` (logger instance) configurable was provided, SDK will use that particular logger for all its logging operations.
-
-If all logging returned by this SDK needs to be disabled, usage of `NullHanlder` provided by the standard `logging` module is advised.  
-
-
-## Custom HTTP client implementation
-
-By default, two HTTP implementations are provided:
-- Synchronous using `requests` HTTP library
-- Asynchronous using `aiohttp` HTTP library
-
-For creating custom HTTP client code, use either `Client` or `ClientAsync` client and inject your transport during initialisation:
```

Comment:
It's nice to have example, maybe you can keep them?

Reply:
done